R/utils.R

```diff
@@ -44,3 +44,7 @@ add_colours_missing_names <- function(names, colour_aliases) {
   }
   colour_aliases
 }
+
+get_hex <- function(palette, label) {
+  hex <- palette %>% filter(.data$label == .env$label) %>% pull (.data$hex)
+}
```

Comment:
This returns invisibly, i.e. `get_hex()` produces no visible output. Is this intentional?